docs/content/stable/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md

```diff
@@ -88,15 +88,15 @@ sudo alternatives --display python3
 python3 -V
 ```
 
-If you are using Python later than v3.6, install the [selinux](https://pypi.org/project/selinux/) package corresponding to your version of python. For example, using [pip](https://pip.pypa.io/en/stable/installation/), you can install as follows:
+Install the [selinux](https://pypi.org/project/selinux/) package corresponding to your version of python. For example, using [pip](https://pip.pypa.io/en/stable/installation/), you can install as follows:
```

Comment:
This can all be left alone for older installations, it's correct. Only improvement would be we also allow 3.9 on the node.